src/domain/application/components/ApplicationSendDialog.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  Button,
+  Dialog,
+  IconCheck,
+  IconQuestionCircle,
+  Notification,
+  TextInput,
+  ToggleButton,
+} from 'hds-react';
+import React, { useEffect } from 'react';
+import { useTranslation } from 'react-i18next';
+import { ApplicationSendData, PaperDecisionReceiver } from '../types/application';
+import { FormProvider, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { sendSchema } from '../yupSchemas';
+import { Box, Grid, GridItem, VisuallyHiddenInput } from '@chakra-ui/react';
+import Text from '../../../common/components/text/Text';
+
+type Props = {
+  isOpen: boolean;
+  isLoading: boolean;
+  onClose: () => void;
+  onSend: (paperDecisionReceiver?: PaperDecisionReceiver | null) => void;
+  applicationId: number;
+};
+
+const ApplicationSendDialog: React.FC<Props> = ({
+  isOpen,
+  isLoading,
+  onClose,
+  onSend,
+  applicationId,
+}) => {
+  const { t } = useTranslation();
+  const formContext = useForm<ApplicationSendData>({
+    resolver: yupResolver(sendSchema),
+    defaultValues: {
+      applicationId: applicationId,
+      orderPaperDecision: false,
+      paperDecisionReceiver: null,
+    },
+  });
+
+  const { handleSubmit, formState, register, watch, setValue, resetField, reset } = formContext;
+  const [orderPaperDecisionChecked] = watch(['orderPaperDecision']);
+  const isConfirmButtonEnabled = formState.isValid;
+  const dialogTitle = t('hakemus:sendDialog:title');
+
+  useEffect(() => {
+    formContext.setValue('applicationId', applicationId);
```

Comment:
Is this needed? I was wondering that can the `applicationId` really change in the lifetime of this component?

Reply:
You're right. I removed this.

Reply:
Removing this breaks the tests. Confirmation button is no longer enabled. It works in browser though. I will investigate...

Reply:
Now it should be fixed.